webviewer/src/pages/Account/components/PageContent/PageContent.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Profile } from '../Profile';
+import { PageWrapper } from '../PageWrapper';
+
+export const PageContent = ({ activeTab, tabs }) => {
+  switch (activeTab) {
+    case tabs[0].name:
```

Comment:
using indexes in such cases is not the best practice. 
maybe to use object with tabKey: tabConfig or something else to use some local ID for the tab. 
and then you will need to use tabs array, you can use Object(TABS).values() and receive the array that you want to map

Reply:
fixed